wiki/Gameplay/Accuracy/es.md

```diff
@@ -17,17 +17,17 @@ En osu!, la precisión se calcula ponderando el [juicio](/wiki/Gameplay/Judgemen
 Referencia para un círculo:
 
 ```
-300 -> 300 / 300 = 1   = 100.00%
-100 -> 100 / 300 = 1/3 =  33.33%
-50  ->  50 / 300 = 1/6 =  16.67%
-0   ->   0 / 300 = 0   =   0.00%
+300 -> 300 / 300 = 1   = 100.00 %
+100 -> 100 / 300 = 1/3 =  33.33 %
+50  ->  50 / 300 = 1/6 =  16.67 %
+0   ->   0 / 300 = 0   =   0.00 %
 ```
 
 ### ![](/wiki/shared/mode/taiko.png) osu!taiko
 
 ![Precisión = (número de GREATs + 0.5 \* número de GOODs) / (número de GREATs + número de GOODs + número de fallos)](img/accuracy_taiko_updated.png "Fórmula de precisión para osu!taiko")
 
-En osu!taiko, la precisión se calcula tomando la suma de la precisión de la nota (qué tan cerca estuvo de tocar la nota a tiempo) dividida por el número total de notas anotadas hasta el momento. Las precisiones de las notas están etiquetadas como GREAT (良) (cuenta como 100%), GOOD (可) (cuenta como 50%) (mitad), y MISS/BAD (不可) (cuenta como 0%, lo que también rompe el combo). Los redobles de tambor y los spinners no influyen en la precisión.
+En osu!taiko, la precisión se calcula tomando la suma de la precisión de la nota (qué tan cerca estuvo de tocar la nota a tiempo) dividida por el número total de notas anotadas hasta el momento. Las precisiones de las notas están etiquetadas como GREAT (良) (cuenta como 100 %), GOOD (可) (cuenta como 50 %) (mitad), y MISS/BAD (不可) (cuenta como 0 %, lo que también rompe el combo). Los redobles de tambor y los spinners no influyen en la precisión.
```

Comment:
No debería ser la precisión de `cada` nota en lugar de `la` nota?

Reply:
In English it says "the note accuracy"